README.md

```diff
@@ -57,6 +63,8 @@ def duplicates?(array)
 end
 ```
 
+ANSWER: O(n**2) - there is a loop within a loop, and each has a big-o of n
```

Comment:
Thanks!

On Tuesday, 13 September 2016, Crystal <notifications@github.com> wrote:

> In README.md
> <https://github.com/Ada-C6/big-o/pull/31#discussion_r78624273>:
>
> > @@ -57,6 +63,8 @@ def duplicates?(array)
> >  end
> >  ```
> >
> > +ANSWER: O(n**2) - there is a loop within a loop, and each has a big-o of n
>
> Just FYI - I wouldn't use ** to represent power, as that's a ruby thing.
> Also syntax for Big O is O(n^2).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ada-C6/big-o/pull/31/files/2547c4c17f7d95f974916512d1223008ad13b3a1#r78624273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQvVsi6jmfvea-TWhtzjtijx5hV__AqPks5qpvTCgaJpZM4J6wZi>
> .
>